### PR TITLE
New dashboard with client & freelancer view. Redesigned Chat popup and other pages to match the figma designs

### PR DESCRIPTION
**Pull request by ssani7**
- Redesigned chat popup in profile, application-preview and application-list and other pages that had this feature. This popup is more minimalist and pops up from the bottom of the page like linkedin

- Made the pages look like figma design for Brief Details and Brief Applications page. Used dummy data for now to populate some fields. Implemented tailwindcss and css modules and removed bootstrap

- New dashboard page. Contains client view, freelancer view and message. Created separate components for client view and freelancer view  that shows list of application and application current state for the client and the freelancer.

### CODE REVIEW

File: api/src/frontend/components/chat-popup.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+interface ChatPopupProps  {
+    showMessageBox:boolean,
+    setShowMessageBox: Function,
+    browsingUser: User | null,
+    targetUser: User | null
+}
+
+const ChatPopup = (props:ChatPopupProps) => {
+    const {showMessageBox,setShowMessageBox, browsingUser,targetUser} = props
+    const chatPopupStyle = {
```
Comment (samelamin) [inline api/src/frontend/components/chat-popup.tsx]: Can we move this to a stylesheet please? I would rather we avoid using styles in tsx files

**Review by samelamin — APPROVED** [review]

File: api/src/frontend/components/chat-popup.tsx (hunk 1 above)
Comment (ssani7) [reply api/src/frontend/components/chat-popup.tsx]: Resolved

File: api/src/frontend/Styles/chat-popup.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    margin: 0;
+}
+
+.MuiBox-root{
+    position: fixed;
+    right: 150px;
+    bottom: 0;
+    width: 400px;
+    height: 480px;
+    background-color: #2c2c2c;
```
Comment (cuteolaf) [inline api/src/frontend/Styles/chat-popup.css:18]: can you create a theme color and replace this hard-coded color?

File: api/src/frontend/Styles/dashboard.css [hunk 3]
```diff
@@ -9,63 +9,125 @@
...
-    }
+.str-chat ::-webkit-scrollbar {
+    width: 3px;
+}
 
-    .chat-list-container{
-        width: 50vw;
-    }
+.str-chat ::-webkit-scrollbar-track {
+    background: #2C2C2C;
```
Comment (cuteolaf) [inline api/src/frontend/Styles/dashboard.css:76]: same

**Review by cuteolaf — APPROVED** [review]
overall, looks good to me.
thanks, @ssani7

File: api/src/frontend/Styles/dashboard.css (hunk 3 above)
Comment (ssani7) [reply api/src/frontend/Styles/dashboard.css:76]: Opps. Forgot to do it here.

File: api/src/frontend/Styles/chat-popup.css (hunk 2 above)
Comment (ssani7) [reply api/src/frontend/Styles/chat-popup.css:18]: Resolved

**Pull request merged**